documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
+    .relationships(vec![
+      MethodRelationship::CapabilityDelegation,
+      MethodRelationship::CapabilityInvocation,
+    ])
+    .apply()
+    .await?;
+```
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+
+```js
+await account.attachMethodRelationships({
+    fragment: "my-next-key",
+    relationships: [
+        MethodRelationship.CapabilityDelegation,
+        MethodRelationship.CapabilityInvocation
+    ]
+})
+```
+</TabItem>
+</Tabs>
+
+This will add `CapabilityDelegation` and `CapabilityInvocation` Relationships to the created Verification Method with the fragment `my-next-key`.
+
+```json
+{
+   "doc":{
+      "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+      "verificationMethod":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z2Zthec5siTfxCjPwZUHGDGybKNy9oc3ZYeftvEE2nEL3"
+         }
+      ],
+      "capabilityDelegation":[
+         "did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key"
+      ],
+      "capabilityInvocation":[
+         {
+            "id":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#sign-0",
+            "controller":"did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf",
+            "type":"Ed25519VerificationKey2018",
+            "publicKeyMultibase":"z5k7vzMVuXXj8MJDcfzP2owvc8xKBA6BBsAkFf1GSNu2X"
+         },
+         "did:iota:6T4PRHWp7bNsKaBWr1gVtUBQfLaxAKqKGAeJWFBZkMyf#my-next-key"
+      ]
+   },
+   "meta":{
+      "created":"2022-04-13T09:27:48Z",
+      "updated":"2022-04-13T09:28:23Z"
+   }
+}
+```
+
+:::note
+Since the added Verification Method has `capabilityInvocation` Relationship, it can be used to sign future Document updates using `PublishOptions`.
+:::
+
+### 4. Adding a Service
+
+"Services are used in DID documents to express ways of communicating with the DID subject or associated entities. 
+A service can be any type of service the DID subject wants to advertise, including decentralized identity management services for further discovery, 
+authentication, authorization, or interaction." [[w3c](https://www.w3.org/TR/did-core/#services)]
+
+Read more about the services in the [standards proposed by W3C](https://www.w3.org/TR/did-core/#services).
+
+The following code adds a service to the DID Document and publishes it to the tangle.
+
+<Tabs>
+<TabItem value="Rust" label="Rust">
+
+```rust
+account
+    .update_identity()
+    .create_service()
+    .fragment("my-service-1")
+    .type_("MyCustomService")
+    .endpoint(Url::parse("https://example.com")?)
+    .apply()
+    .await?;
+```
+
+</TabItem>
+<TabItem value="node" label="Node.js">
+
+```js
+await account.createService({
+   fragment: "my-service-1",
+   type: "MyCustomService",
+   endpoint: "https://example.com"
+})
+```
+</TabItem>
+</Tabs>
+
+:::note
```

Comment:
A bit too many notes / tips here. Perhaps just make it a paragraph to explain the information.